hack/make-devenv.sh

```diff
@@ -31,7 +31,7 @@ run_builder() {
 
 run() {
     docker run --rm --privileged -w /melage -v /var/run/docker.sock:/var/run/docker.sock \
-        -v $(pwd):/melage -ti ${IMAGE_TAG}:latest hack/make-devenv.sh setup
+        -v "$(pwd):/melage" -ti ${IMAGE_TAG}:latest hack/make-devenv.sh setup
```

Comment:
As above, and also lets fix the typo since we are here:

```suggestion
        -v "$(pwd)":/melange -ti ${IMAGE_TAG}:latest hack/make-devenv.sh setup
```